modules/gdnative/videodecoder/register_types.cpp

```diff
@@ -0,0 +1,17 @@
+/* register_types.h */
```

Comment:
For consistency, this file and its header should also use the Godot copyright header. They also lack a newline at end of file.